applications/topsApp.py

```diff
@@ -519,6 +519,14 @@
     mandatory=False,
     doc='')
 
+ION_MASKED_AREAS = Application.Parameter('ION_maskedAreas',
```

Comment:
You added a very clear explanation of this added parameter to topsApp.xml. I would actually add that comment here.

Reply:
OK, I also copied the comments here.